README.md

```diff
@@ -1 +1,18 @@
-# ah-django-frontend
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
```

Comment:
Use your own words to describe your project, how to run it... 
what do you run to start the project... Make it easy for a 5-year-old 😁